packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -117,7 +118,7 @@ function initializeFlow(projectPath, flowconfig, otherFlowTypedDefs) {
         Promise.reject(new Error(
           'Flow integration was disabled because the global Flow version does not match.\n' +
           'You may either remove the global Flow installation or install a compatible version:\n' +
-          '  npm install -g flow-bin@' + localVersion
+          chalk.cyan('  npm install -g flow-bin@' + localVersion)
```

Comment:
Let's color just the `npm` word.
Like [here](https://github.com/facebookincubator/create-react-app/blob/ab3913cc113b2392fa0329f699571bae3c213308/packages/react-scripts/scripts/build.js#L177).